dacbench/benchmarks/toysgd_benchmark.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+import numpy as np
+import pandas as pd
+from gym import spaces
+
+from dacbench.abstract_benchmark import AbstractBenchmark, objdict
+import importlib
+import dacbench.envs.toysgd
+importlib.reload(dacbench.envs.toysgd)
+
+HISTORY_LENGTH = 40
+
+INFO = {
+    "identifier": "toy_sgd",
+    "name": "Learning Rate and Momentum Adaption for SGD on Toy Functions",
+    "reward": "Negative Log Regret",
+    "state_description": [
+        "Remaining Budget",
+        "Gradient",
+        "Current Learning Rate",
+        "Current Momentum"
+    ],
+}
+
+DEFAULTS = objdict(
+    {
+        "action_space_class": "Box",
+        "action_space_args": [-np.inf * np.ones((2,)), np.inf * np.ones((2,))],
+        "observation_space_class": "Dict",
```

Comment:
I'll not say you shouldn't or can't do that, but Dict spaces are impractical without the dict wrapper. If you don't care, feel free to ignore

Reply:
I just took inspiration from the sgd benchmark. If that's suboptimal, feel free to remodel it :) or I could also do it tomorrow